concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -262,38 +267,35 @@ def main():
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         headers = {
-            'Content-Type': 'application/octet-stream',
+            'Content-Type'             : 'application/octet-stream',
             'concent-client-public-key': b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
-        }
+        },
+        expected_status = 200
     )
 
     # Test CASE 2D + 3 + 4B + 5. Requestor sends ForceSubtaskResultsResponse with SubtaskResultsRejected
+    task_2 = create_task(task_to_compute_timestamp, task_id + '2D', compute_task_def_deadline)
+
     api_request(
         cluster_url,
         'send',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         force_subtask_results(
-            timestamp = timestamp_to_isoformat(current_time),
-            ack_report_computed_task = ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 14401),  # current_time - 04:00:01
-                subtask_id = task_id + '2D',
-                task_to_compute = task_to_compute(
-                    timestamp = timestamp_to_isoformat(current_time - 14402),  # current_time - 04:00:02
-                    compute_task_def = compute_task_def(
-                        task_id     = task_id + '2D',
-                        subtask_id  = task_id + '2D',
-                        deadline    = current_time + 3600,  # current_time + 01:00:00
-                    )
+                timestamp                   = current_timestamp,
+                ack_report_computed_task    = ack_report_computed_task(
+                        timestamp           = ack_report_computed_task_timestamp,  # current_time - 04:00:10
+                        subtask_id          = task_id + '2D',
+                        task_to_compute     = task_2
                 )
-            )
         ),
         headers = {
-            'Content-Type':                     'application/octet-stream',
-            'concent-client-public-key':        b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
-            'concent-other-party-public-key':   b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii'),
-            'temporary-account-funds':          'True'
-        }
+            'Content-Type'                  : 'application/octet-stream',
+            'concent-client-public-key'     : b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
```

Comment:
Ekhm... our style is not that eccentric :) I don't know where it's coming from but it was fine before this change.